py/rcm_recoverer.py

```diff
@@ -37,43 +36,34 @@ def smart_open(filename, mode="r"):
             fh.close()
 
 
-def generate_rcm(reads_file_name, compressed_file_name, cliques_ids_file_name,
-                 out_file):
-    # Obtain read ids
-    with smart_open(reads_file_name, "r") as fh:
-        ids = [str(record.id) for record in SeqIO.parse(fh, "fasta")]
+def generate_rcm(compressed_rcm_file_name, clique_ids_file_name, out_file):
```

Comment:
Unfortunately this file remains not rcm one. I didn't convert it since DSF doesn't take any fast file as an input and probably it's ok. So the argument doesn't change its meaning.